src/domain/harbors/HarborsList.tsx

```diff
@@ -5,14 +5,16 @@ import { useTranslation } from 'react-i18next';
 import Table, { Column } from '../../common/table/Table';
 import { HarborData } from './utils';
 import {
-  IconPole,
   IconFence,
   IconTrash,
   IconWaterTap,
+  IconPlug,
+  IconStreetLight,
 } from '../../common/icons';
 import HarborDetails from './harborDetails/HarborDetails';
 import InternalLink from '../../common/internalLink/InternalLink';
-import styles from '../../common/icons/icon.module.scss';
+import styles from './harborsPage.module.scss';
```

Comment:
These `styles` appear to be imported but not applied anywhere in this module!

Reply:
https://github.com/City-of-Helsinki/berth-reservations-admin/blob/c65757f98a0e327717327fdd748b1504c0f16be5/src/domain/harbors/HarborsList.tsx#L161

They are used here. This fixes the styling issues related to margins and background color.

Reply:
Ah, I see.